### ISSUE TITLE
same chunk names and hashes on different css files

### ISSUE THREAD

**Issue by n1crack** [JeffreyWay/laravel-mix#368]
- Laravel Mix Version: "laravel-mix": "^0.7.4",
- Node Version (`node -v`):  v7.5.0
- NPM Version (`npm -v`): 4.1.2
- OS: macOS 10.12.3

### Description:
I am getting same chunk names and hashes on different css files.

### Steps To Reproduce:

this is my webpack.mix.js file : 

````
const {mix} = require('laravel-mix');

mix.webpackConfig({
    resolve: {
        alias :  {
            'jquery-ui/sortable' : 'jquery-ui-browserify/ui/jquery.ui.sortable.js'
        }
    }
});


// Site JS + CSS
mix.js('resources/assets/js/site.js', 'public/js')
    .sass('resources/assets/sass/site/site.scss', 'public/css');

// Admin JS + CSS
mix.js('resources/assets/js/admin.js', 'public/js')
    .sass('resources/assets/sass/admin/admin.scss', 'public/css');

mix.version();

mix.copy('resources/assets/js/admin/ga-picker.js', 'public/js');
````

and this is what I get : 

````
 DONE  Compiled successfully in 17384ms

                                                                    Asset       Size  Chunks                    Chunk Names
             vendor_images/sort_both.png?9a6486086d09bb38cf66a57cc559ade3  201 bytes          [emitted]
           fonts/fontawesome-webfont.eot?674f50d287a8c48dc19ba404d20fe713     166 kB          [emitted]
           fonts/fontawesome-webfont.ttf?b06871f281fee6b241d60582ae9369b9     166 kB          [emitted]
          fonts/fontawesome-webfont.woff?fee66e712a8a08eef5805a46892932ad      98 kB          [emitted]
         fonts/fontawesome-webfont.woff2?af7ae505a9eed503f8b8e6982036873e    77.2 kB          [emitted]
                 vendor_images/alpha.png?5278808a4d6b13c98d9484109547b749  488 bytes          [emitted]
  fonts/glyphicons-halflings-regular.eot?f4769f9bdb7466be65088239c12046d1    20.1 kB          [emitted]
      vendor_images/alpha-horizontal.png?1b3c0ada204c72f99386a99545a5ccfc  557 bytes          [emitted]
        vendor_images/hue-horizontal.png?34a3768e23e11bc227fa6dd3091ff26e  478 bytes          [emitted]
                   vendor_images/hue.png?e75b2a9cae62d4c2951a9fea86e8fac4  504 bytes          [emitted]
            vendor_images/saturation.png?668d0a0e7a0189f62898bcc57ab8370c    4.14 kB          [emitted]
              vendor_images/sort_asc.png?9326ad44ae4bebdedd141e7a53c2a730  160 bytes          [emitted]
     vendor_images/sort_asc_disabled.png?d7dc10c78f23615d328581aebcd805eb  148 bytes          [emitted]
           fonts/fontawesome-webfont.svg?912ec66d7572ff821749319396470bde     444 kB          [emitted]  [big]
             vendor_images/sort_desc.png?1fc418e33fd5a687290258b23fac4e98  158 bytes          [emitted]
    vendor_images/sort_desc_disabled.png?bda51e15154a18257b4f955a222fd66f  146 bytes          [emitted]
  fonts/glyphicons-halflings-regular.svg?89889688147bd7575d6327160d64e760     109 kB          [emitted]
  fonts/glyphicons-halflings-regular.ttf?e18bbf611f2a2e43afc071aa2f4e1512    45.4 kB          [emitted]
 fonts/glyphicons-halflings-regular.woff?fa2772327f55d8198301fdb8bcfc8158    23.4 kB          [emitted]
fonts/glyphicons-halflings-regular.woff2?448c34a56d699c29117adc64c43affeb      18 kB          [emitted]
                                        /js/admin.36eb8e85bf6fdd5b72de.js     1.6 MB       0  [emitted]  [big]  /js/admin
                                         /js/site.7bb779218af559e807a1.js     585 kB       1  [emitted]  [big]  /js/site
                                       /css/site.fbfa5c51bb950bb1fa44.css     957 kB       1  [emitted]  [big]  /js/site
                                      /css/admin.fbfa5c51bb950bb1fa44.css     1.1 MB       1  [emitted]  [big]  /js/site
                                                        mix-manifest.json  226 bytes          [emitted]
                                                          js/ga-picker.js    2.61 kB          [emitted]
````

for site.css and admin.css, hashes are same as well as their chunk names. 

 /css/site.fbfa5c51bb950bb1fa44.css
/css/admin.fbfa5c51bb950bb1fa44.css

**Comment by juaniiton1** [comment]
any clue?

**Comment by JeffreyWay** [comment]
Fixed in #674.
